packages/scandipwa/src/store/Review/Review.dispatcher.js

```diff
@@ -42,15 +42,9 @@ export class ReviewDispatcher {
         };
     }
 
-    submitProductReview(dispatch, options) {
-        return fetchMutation(ReviewQuery.getAddProductReviewMutation(
-            this.prepareReviewData(options)
-        )).then(
-            /** @namespace Store/Review/Dispatcher/ReviewDispatcher/submitProductReview/fetchMutation/then/dispatch */
-            () => dispatch(showNotification('success', 'You submitted your review for moderation.')),
-            /** @namespace Store/Review/Dispatcher/ReviewDispatcher/submitProductReview/fetchMutation/then/dispatch/catch */
-            () => dispatch(showNotification('error', __('Error submitting review!')))
-        );
+    async submitProductReview(dispatch, options) {
+        await fetchMutation(ReviewQuery.getAddProductReviewMutation(this.prepareReviewData(options)));
```

Comment:
I think this await should be wrapped into try/catch, as before the change there was catch processing, and now it's not there.